e2e/test-app-v10-zone-v11/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "test-app-v10",
```

Comment:
NIT, this should be `test-app-v10-zone-v11`

Reply:
Got it, updated.